src/CoreText/CTStringAttributes.cs

```diff
@@ -294,10 +301,17 @@ public CTStringAttributes (NSDictionary dictionary)
 			}
 		}
 
-		[iOS (10,0)][Mac(10,12)]
+		[iOS (10,0)][Mac (10,12)]
 		public int? HorizontalInVerticalForms {
-			get {return Adapter.GetInt32Value (Dictionary, CTStringAttributeKey.HorizontalInVerticalForms);}
-			set {Adapter.SetValue (Dictionary, CTStringAttributeKey.HorizontalInVerticalForms, value);}
+			get {
+				var x = CTStringAttributeKey.HorizontalInVerticalForms;
+				return x != null ? Adapter.GetInt32Value (Dictionary, x) : new int? ();
```

Comment:
Never mind looks like it was an other issue... I just thought `new int? ();` fixed it. It's confusing, I really tried multiple times, likely didn't clean properly.
All good now 👍